### DISCUSSION TITLE
Use Translation Strings in Collection Name Translations

### DISCUSSION THREAD

**Discussion by obezzad** [directus/directus#13956] [Feature Requests]
Similarly to #12868, but this applies to collection naming:
![image](https://user-images.githubusercontent.com/2561446/174279973-ad581df4-49b5-4331-afa1-471ec9e04dea.png)

Such feature would be beneficial to avoid repetitive translations, especially for fields such as `title`. Re-use the string elsewhere to refer to the collection. Also, correcting a translation is a pain without a single source of truth.

**Comment by azrikahar** [comment]
Yeah although the discussion title is regarding fields, collections are also taken into account (and should indeed be included when this gets implemented) as seen in this comment in that discussion: https://github.com/directus/directus/discussions/12868#discussioncomment-2599007 👍

**Reply by obezzad** [reply]
I see. Thank you for the prompt response. Is there any pending PR I can contribute to?

**Reply by azrikahar** [reply]
There is no pending PR yet 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
